front/src/modules/ui/board/components/BoardColumnEditTitleMenu.tsx

```diff
@@ -57,7 +47,10 @@ export const COLOR_OPTIONS = [
   { name: 'Orange', id: 'orange' },
   { name: 'Yellow', id: 'yellow' },
   { name: 'Gray', id: 'gray' },
-];
+] as {
```

Comment:
do we need to use `as` keyword here we should be able to use COLOR_OPTIONS: { name: string, id: ThemeColor }